### PR TITLE
Update SUSHI --init to avoid publisher errors

### PR DESCRIPTION
**Pull request by ngfreiter**
Fixes https://github.com/FHIR/sushi/issues/884.

`sushi --init` was causing errors and warnings in the IG Publisher. These turned out to be related to:
* Not having any examples of the `MyPatient` profile
* Not having any description on the `MyPatient` profile
* Not having a `publisher` set

So to fix this, this PR updates the `patient.fsh` file to have an example and a description. It also updates the default `sushi-config.yaml` file to contain default values for `publisher.name` and `publisher.url`, and updates the `init()` function so that the user can specify custom values for these fields.

# Testing
To test this, try doing `sushi --init` on `master`, and build with the publisher with `_genonce`. The QA page, `output/qa.html`, will have several errors and warnings. Then do the same on this branch, and the QA page should look clean, like this: 
image

The QA page should definitely be clean if you use default values, and should likely be clean if you use custom values, but note that certain custom values could lead to warnings. But either way make sure to test both approaches. Ensure that the publisher value you use is inserted in the footer of pages as shown here:
image

### CODE REVIEW

**Review by mint-thompson — DISMISSED** [review]
Tried it out on my windows machine, and everything appears to be in good order. Thank you!

File: src/utils/Processing.ts [hunk 1]
```diff
@@ -429,6 +429,19 @@ export async function init(): Promise<void> {
       configDoc.set(field, userValue);
     }
   });
+
+  // And for nested publisher fields
+  ['name', 'url'].forEach(field => {
+    const userValue = readlineSync.question(
+      `Publisher ${field.charAt(0).toUpperCase() + field.slice(1)} (Default: ${configDoc
```
Comment (jafeltra) [inline src/utils/Processing.ts:436]: I'm not sure how much this matters, but I think you can alternatively use lodash's `upperFirst` function to do the same as this string concatenation.

**Review by jafeltra — COMMENTED** [review]
This looks good! I confirmed the errors are resolved when I use the defaults and are the expected errors when I use my own values (my url did not resolve). I just had one small comment that you can take or leave.

File: src/utils/Processing.ts (hunk 1 above)
Comment (ngfreiter) [reply src/utils/Processing.ts]: Good point, done in https://github.com/FHIR/sushi/pull/899/commits/b06aaf24c6f8cf172712947988bba9e8246ed902.

**Review by jafeltra — APPROVED** [review]
Looks good to me!

**Review by mint-thompson — APPROVED** [review]

**Pull request merged**